### PR TITLE
PyCall → PythonCall

### PR DESCRIPTION
**Pull request by Alexsp32**
Update:

I've decided to move this to [a separate package](https://github.com/NQCD/MACEModels.jl) instead to avoid requiring Python in NQCModels.jl.

Instead, this PR now makes the ASE and ASE friction interfaces usable with PythonCall.jl to fit in the structure of using PythonCall and CondaPkg.jl for packaging Python dependencies.

---

Avoid ase calculator and as many Julia -> Python -> Torch pipelines as reasonably possible without re-writing MACE for Julia.

### CODE REVIEW

**Comment by Alexsp32** [comment]
I've decided to move this to [a separate package](https://github.com/NQCD/MACEModels.jl) instead to avoid requiring Python in NQCModels.jl. 

Instead, this PR now makes the ASE and ASE friction interfaces usable with PythonCall.jl to fit in the structure of using PythonCall and CondaPkg.jl for packaging Python dependencies.

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -42,7 +42,12 @@ jobs:
         run: julia -e 'using Pkg; Pkg.Registry.add(RegistrySpec(url="https://github.com/JuliaMolSim/MolSim"))'
 
       - uses: julia-actions/julia-buildpkg@v1
+        env:
+          LD_LIBRARY_PATH: $HOME/libxc-5.1.5/lib
```
Comment (reinimaurer1) [inline .github/workflows/CI.yml:46]: Why is there a libxc dependency here? this only seems to be in the CI. does anything in NQCModels or its tests require that?

File: test/ase_pycall.jl [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    using PyCall
+
+    ase = pyimport("ase")
+
+    h2 = ase.Atoms("H2", [(0, 0, 0), (0, 0, 0.74)])
+    h2.center(vacuum=2.5)
+
+    @testset "EMT" begin
+        emt = pyimport("ase.calculators.emt")
+        h2.calc = emt.EMT()
```
Comment (reinimaurer1) [inline test/ase_pycall.jl:12]: just a note: it used to be that calculators require setting with atoms.set_calculator(calc). EMT might be a rare exception where you can directly set it like this

**Review by reinimaurer1 — APPROVED** [review]
all great. I only left some comments

File: .github/workflows/CI.yml (hunk 1 above)
Comment (Alexsp32) [reply .github/workflows/CI.yml:46]: test/ase_pycall.jl:17-22 uses it to test the ASE calculator interface with GPAW. I couldn't find a working conda package for GPAW, or Id've changed the test mechanism to install it through CondaPkg.jl and make the tests work on any machine without having to set up ase and GPAW first.

**Pull request merged**